code/orders-api-rest/api/openapi-rest.yml

```
@@ -816,48 +816,179 @@ paths:
-        - UserPersonalCabinet
-      summary: Get user info.
-      description: Get main user info such as Name, Surname, Email etc for the registered user.
-      operationId: getPersonalUserInfo
-      security:
-        - bearerAuth: [ ]
+        - PasswordResetController
+      summary: Request password reset
+      description: |
+        Accepts an email and returns 200 status regardless of whether the email exists.
+        This prevents email enumeration attacks while maintaining user experience.
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)